region/maridia/outer.json

```diff
@@ -3129,15 +3431,65 @@
                     "If you lose the fast crab, it is always possible to get it back by climbing the peak to the right. The crab there can be lured with HiJump alone."
                   ],
                   "devNote": "Although this may go through 7 and 8, it requires starting at 2 to get the crab to move into position."
+                },
+                {
+                  "name": "Cross Room Jump - Speed Jump",
+                  "notable": false,
+                  "requires": [
+                    "canCrossRoomJumpIntoWater",
+                    {"adjacentJumpway": {
+                      "jumpwayType": "platformBelow",
+                      "fromNode": 2,
+                      "maxHeight": 10,
+                      "maxLeftPosition": -38.5,
+                      "minRightPosition": 0
+                    }},
+                    "SpeedBooster"
+                  ],
+                  "note": [
+                    "Run and jump up through the door using SpeedBooster in the room below.",
+                    "Turn to the left immediately after jumping, to allow running further and to cancel the rightward momentum before passing through the transition."
+                  ]
                 }
               ],
-              "devNote": "One-way link for superless crab climb."
+              "devNote": "One-way link for superless crab climb and cross-room jump."
             }
           ]
         },
         {
           "from": 3,
           "to": [
+            {
+              "id": 4,
+              "strats": [
+                {
+                  "name": "Cross Room Jump - Speed Jump",
+                  "notable": false,
+                  "requires": [
+                    "canCrossRoomJumpIntoWater",
+                    {"adjacentJumpway": {
+                      "jumpwayType": "platformBelow",
+                      "fromNode": 2,
+                      "maxHeight": 10,
```

Comment:
It's a very particular jump in PCJR.  Between hitting the crab, and gaining too much height.  Wiggle Tech helps a lot.  Not using the full runway helps some too.

Reply:
Croc room is suspected of needing wiggle tech too :octocat:

Reply:
Agreed, added a note suggesting not using the full runway. Also mentioned wiggling as an option: it's not required in either room but does seem to make it easier